packages/zowe-explorer-ftp-extension/README.md

```diff
@@ -12,6 +12,29 @@ Enhancements:
 
 For more information, see [Changelog](https://github.com/zowe/vscode-extension-for-zowe/blob/master/packages/zowe-explorer-ftp-extension/CHANGELOG.md).
 
+## Prerequisites
+
+Host-side prerequisites:
+
+- Obtain remote access to z/OS FTP service.
+
+  **Follow these steps:**
+
+  1. Connect to z/OS with the FTP client.
+  1. Run the following command in the FTP client:
```

Comment:
Looks good, but every step is numbered **1.**

So steps should go down like:
1.
2.
3.

Reply:
@katelynienaber Thanks for the review! This was intentional, as markdown automatically numbers items in an ordered list for you if you just number everything with `1. `. It's quite handy in case you ever need to add a step higher up in the list - then you don't need to manually fix the numbering for all the steps further down. It looks like this in the rendered file:

<img width="489" alt="Screen Shot 2021-03-24 at 5 25 51 AM" src="https://user-images.githubusercontent.com/45975633/112286534-cc2ab780-8c61-11eb-82c3-8bb1d9d29301.png">

However, if you still prefer, I can change to manual numbering since that will also display correctly.